src/devices/uFire_ISE/UFirePh .cs

```diff
@@ -36,14 +36,14 @@ public UFirePh(I2cDevice i2cDevice)
         }
 
         /// <summary>
-        /// Tries to measurer pH (Power of Hydrogen) .
+        /// Tries to measure pH (Power of Hydrogen) .
         /// </summary>
         /// <param name="pH">The measure pH value</param>
         /// <param name="temp">Temperature compensation is available by passing the temperature.</param>
         /// <returns>True if it could measure pH (Power of Hydrogen) else false</returns>
         public bool TryMeasurepH(out float pH, Temperature? temp = null)
         {
-            ElectricPotential mV = MeasuremV();
+            ElectricPotential mV = Measure();
             if (mV.Value == -1)
             {
                 pH = -1;
```

Comment:
next line not needed, and remove all the double pH = -1. Or maybe you want to use the property, so then you should consider:
```csharp
Ph = pH;
```

Reply:
Also, are you sure that -1 is not a correct value returned by Measure?

Reply:
yes I am sure it return -1 as a value if it get an error.